docs/articles/guides/custom-icons.md

```diff
@@ -0,0 +1,100 @@
+# Register Icon pack
+
+Nebular comes with an `<nb-icon></nb-icon>` component that accepts an `icon="icon-name"` and `pack="icon-pack"` parameters.
+By default, Nebular includes only small and limited pack called `nebular-essentials`. This pack consists of essential icons, such as `close-outline`, `checkmark-outline`, etc that are required by the Nebular components (modals, accordions, etc).
+
+## Eva Icons
+
+In case you need to have more icons available, Nebular provides a `@nebular/eva-icons` pack with 480+ SVG icons.
+
+Install the pack:
+
+ ```sh
+ npm i @nebular/eva-icons@next
+ ```
+
+This command will install Eva Icons pack. Then register `NbEvaIconsModule` into your app module:
+```ts
+import { NbEvaIconsModule } from '@nebular/eva-icons';
+
+@NgModule({
+  imports: [
+    // ...
+    NbEvaIconsModule,
+  ],
+})
+export class AppModule { }  
+```
+Import `NbIconModule` into your feature module where you need to show the icon:
+```ts
+@NgModule({
+  imports: [
+    // ...
+    NbIconModule,
+  ],
+})
+export class PageModule { }
+```
+
+Importing `NbEvaIconsModule` also sets the Eva pack as the default pack, which means you can render a Eva icon without specifying the pack implicitly:
+
+```html
+<nb-icon icon="star"></nb-icon>
+```
+
+## 3rd-party/Custom CSS Pack
+
+In case you need to use any other 3rd party CSS Icon pack, you can easily register it using `NbIconLibraries` service.
+
+First, import css pack files the preferred way (usually in your `angular.json` file).
+Then simply register the pack using `NbIconLibraries` service in you `app.component`:
+
+```ts
+  import { NbIconLibraries } from '@nebular/theme';
+
+  constructor(private iconLibraries: NbIconLibraries) {
+    this.iconLibraries.registerFontPack('font-awesome', { iconClassPrefix: 'fa' });
+    this.iconLibraries.setDefaultPack('nebular');
```

Comment:
I find default pack step a bit confusing. I think this line can be removed since the process of setting default pack described below.